src/libraries/System.Diagnostics.DiagnosticSource/src/System.Diagnostics.DiagnosticSource.csproj

```diff
@@ -85,6 +85,7 @@ System.Diagnostics.DiagnosticSource</PackageDescription>
     <Compile Include="$(CommonPath)Internal\Padding.cs" Link="Common\Internal\Padding.cs" />
     <Compile Include="$(CommonPath)System\HexConverter.cs" Link="Common\System\HexConverter.cs" />
     <Compile Include="$(CommonPath)System\Diagnostics\DiagnosticsHelper.cs" Link="Common\System\Diagnostics\DiagnosticsHelper.cs" />
+    <Content Include="ILLink\ILLink.Descriptors.LibraryBuild.xml" />
```

Comment:
><Content Include="ILLink\ILLink.Descriptors.LibraryBuild.xml" /> [](http://example.com/codeflow?start=4&length=64)

nit: would it be better moving this to line 27?

Reply:
sure, I think Visual Studio automatically added it there when I edited the UI